cmd/nerdctl/build.go

```diff
@@ -360,7 +363,16 @@ func generateBuildctlArgs(cmd *cobra.Command, buildkitHost string, platform, arg
 		return "", nil, false, "", nil, nil, err
 	}
 	if _, err := os.Lstat(filepath.Join(absDir, file)); err != nil {
-		return "", nil, false, "", nil, nil, err
+		if file == buildkitutil.DefaultDockerfileName {
+			logrus.Infof("%s not found, checking if Containerfile exists and overriding it", buildkitutil.DefaultDockerfileName)
```

Comment:
This should be printed only when actually using Containerfile

Reply:
Added a notice and printed to stdout